api/src/tests/multisig_transactions_test.rs

```diff
@@ -343,13 +343,17 @@ async fn test_multisig_transaction_simulation() {
         .create_multisig_account(
             owner_account_1,
             vec![owner_account_2.address(), owner_account_3.address()],
-            2,    /* 2-of-3 */
+            1,    /* 1-of-3 */
             1000, /* initial balance */
         )
         .await;
 
-    // Should be able to simulate the multisig tx without having enough approvals or the transaction
-    // created.
+    let multisig_payload = construct_multisig_txn_transfer_payload(owner_account_1.address(), 1000);
+    context
```

Comment:
can we rename `create_multisig_transaction` into `execute_...` - was quite confused by the name and the fact that it does not return anything